### DISCUSSION TITLE
> Hey, thanks for reaching out. We're aware, the fix will be available in a few days. Meanwhile, you can set the env variable `timestamp.format` to `DD.MM.YYYY HH:mm:ss`.

### DISCUSSION THREAD

**Discussion by Iceforestik** [provectus/kafka-ui#3103] [Q&A] [answered]
> Hey, thanks for reaching out. We're aware, the fix will be available in a few days. Meanwhile, you can set the env variable `timestamp.format` to `DD.MM.YYYY HH:mm:ss`.

@Haarolean I don't understand how to do this, can you write in steps? what is the name of the variable itself and how to export it, this is done through the UI or inside the docker container or at startup
I tried docker exec
timestamp.format=DD.MM.YYYY HH:mm:ss
timestamp_format=DD.MM.YYYY HH:mm:ss
TIMESTAMP_FORMAT=DD.MM.YYYY HH:mm:ss
TIMESTAMP_FORMAT="DD.MM.YYYY HH:mm:ss"

_Originally posted by @Iceforestik in https://github.com/provectus/kafka-ui/issues/2716#issuecomment-1350666149_

**Comment by Haarolean** [comment, marked as answer]
**UPD: we've dropped these properties, now you'll have it displayed automatically based on your browser locale.**
---

@Iceforestik hey, thanks for reaching out.

It's an env. variable, just like the others. 
It depends on the way you pass config properties -- either docker-compose, docker run or other ways like k8s configmap.

**Reply by Iceforestik** [reply]
@Haarolean hello, I am sorry, but I try more various combination
For example:
docker run -p 8080:8080 \
	-e KAFKA_CLUSTERS_0_NAME=local \
	-e KAFKA_CLUSTERS_0_BOOTSTRAPSERVERS=kafka:9092 \
        -e TIMESTAMP_FORMAT="DD.MM.YYYY HH:mm:ss"
	-d provectuslabs/kafka-ui:latest
And I wrote variables as I indicated above and it didn't work once. Of course, I solved the problem using your master image, but I would like to understand how to write this env correctly. It is also not specified in the table https://github.com/provectus/kafka-ui#-environment-variables

**Reply by Haarolean** [reply]
@Iceforestik hey, please update to the latest version, we've dropped these properties, now you'll have it displayed automatically based on your browser locale.
